### ISSUE TITLE
package management

### ISSUE THREAD

**Issue by borisdiakur** [jasonknight/woocommerce-json-api#68]
tl;dr: version control is for source, not builds. Use npm.

Updating the plugin via git pull is risky because you (as a plugin user) never know if braking changes have been introduced since the last pull. Some sort of package management would be really nice. You could for instance use [npm](https://www.npmjs.com/) (which I highly recommend!). Setup a [package.json](https://docs.npmjs.com/files/package.json), [publish](https://docs.npmjs.com/cli/publish) your package and follow the [semantic versioning](https://docs.npmjs.com/misc/semver) principle. In addition to that a [CHANGELOG.md](http://keepachangelog.com/) would be great.

If you need any help, let me know.

**Comment by jasonknight** [comment]
"If you need any help, let me know."

Sure. Do it. I'll merge your changes

**Comment by borisdiakur** [comment]
Whoops, totally forgot about it! Here you go: #69 :walking:

**Comment by borisdiakur** [comment]
Note:

```
$ npm view woocommerce-json-api
npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "view" "woocommerce-json-api" "versions"
npm ERR! node v4.2.6
npm ERR! npm  v2.14.12
npm ERR! code E404

npm ERR! 404 no such package available : woocommerce-json-api
npm ERR! 404 
npm ERR! 404 'woocommerce-json-api' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
:
```

**Comment by jasonknight** [comment]
Yeah, I have been a little ill lately, in and out of the clinic. I will look into registering the project for NPM today and maybe it will start working. I don't use NPM though, so may take a bit of reading.
